src/core/lib/security/credentials/composite/composite_credentials.h

```diff
@@ -21,6 +21,8 @@
 
 #include <grpc/support/port_platform.h>
 
+#include <string.h>
```

Comment:
I believe you should use `#include <string>` throughout the files for `std::string`. If you need to use the C-string things, then you should use `<cstring>`.